README.md

```diff
@@ -1,10 +1,8 @@
-**The Gizra Way Definitive Guide**
-
-*The Gizra Way* to write *The Gizra Way GitBook*. Some guidelines:
-
-1. Document the **bottom line**. Include only processes that have been tested and work. Cut out fluff.
-2. **Iterate**. If this document is not a live document, its useless.
-3. For every standard provide a way, preferably automatic, to **monitor** it.
+# The Gizra Way Definitive Guide
 
+This book is the definitive guide to the Gizra Way, and it tries to encapsulate both the technical
+aspects and our culture.
 
+The idea is a place for everyone to go - to better understand what is expected of them and the processes that encapsulate that. To effectively do this, we document only processes that have been tested and work. We try our best to cut out fluff and give the bottom line.
```

Comment:
The fact that this book is open to everyone does not meant it is written for everyone. It is important to be specific who this book is meant for. What would you feel comfortable replacing "Everyone" with? Gizra employees?